### PR TITLE
Fix readme

### PR DESCRIPTION
**Pull request by Szymiks**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -187,8 +187,8 @@ from .development import *
 ```
 If your database configuration differs from the defaults, you may need to tweak the values below and add them to your local_settings.py too:
 ```
-DATABASES['NAME'] = 'sheetstorm'
-DATABASES['USER'] = 'postgres'
-DATABASES['PASSWORD'] = ''
+DATABASES['default']['NAME'] = 'sheetstorm'
+DATABASES['default']['USER'] = 'postgres'
+DATABASES['deafult']['PASSWORD'] = ''
```
[the remark this answers is not in the document]
Comment (maciejSamerdak) [reply README.md]: Also, @Karrp told me you skipped the **local configuration** part. Can't really say I blame you. But at this moment I don't have any idea how to improve it. Should it be something like 'Refer to local configuration for remaining setup' at the end of all instructions, or maybe some sort of index at the beginning? I wonder if such thing can somehow be generated, or do we have to write it ourselves?

If you can, you could think in the meantime of someway to improve the readability, so the next person doesn't skip the local configuration.

**Comment by maciejSamerdak** [comment]
Also, @Karrp told me you skipped the **local configuration** part. Can't really say I blame you. But at this moment I don't have any idea how to improve it. Should it be something like 'Refer to local configuration for remaining setup' at the end of all instructions, or maybe some sort of index at the beginning? I wonder if such thing can somehow be generated, or do we have to write it ourselves?

If you can, you could think in the meantime of someway to improve the readability, so the next person doesn't skip the local configuration.

File: README.md [hunk 2]
```diff
@@ -108,7 +108,7 @@ git clone https://github.com/Code-Poets/sheetstorm.git
     initdb /usr/local/var/postgres
     ```
 
-8.  Edit file located in */usr/local/var/postgres/pg_hba.conf*, change ***\<method\>*** to ***trust*** in the following line:
+8.  Edit file located in */usr/local/var/postgres/pg_hba.conf* - using administrative rights is needed, change ***\<method\>*** to ***trust*** in the following line:
```
[the remark this answers is not in the document]
Comment (Szymiks) [reply README.md]: Done

File: README.md [hunk 3]
```diff
@@ -49,7 +49,7 @@ git clone https://github.com/Code-Poets/sheetstorm.git
     sudo apt install postgresql postgresql-contrib
     ```
 
-6. Edit file located in */etc/postgresql/<version>/main/pg_hba.conf*, change ***\<method\>*** to ***trust*** in the following line:
+6. Edit file located in */etc/postgresql/<version>/main/pg_hba.conf* - using administrative rights is needed, change ***\<method\>*** to ***trust*** in the following line:
```
[the remark this answers is not in the document]
Comment (Szymiks) [reply README.md]: Done

File: README.md [hunk 4]
```diff
@@ -2,8 +2,8 @@
 Base application for management of employees working hours
 
 ## Software requirements
-Python 3.7.0
-Django 2.1.1
+Python 3.7.0 <br/>
+Django 2.1.1 <br/>
```
[the remark this answers is not in the document]
Comment (Szymiks) [reply README.md]: Done

**Comment by maciejSamerdak** [comment]
I added minor styling fix in the *local configuration* note. You can go ahead and merge it. Don't forget to squash all the fixups first.

File: README.md [hunk 5]
```diff
@@ -13,8 +13,9 @@ Clone following repository:
 git clone https://github.com/Code-Poets/sheetstorm.git
 ```
 
+Note: `Upon completing all steps for required OS, please refer to the local configuration sub-section.`
 
-
+Go to: [Local configuration](#local-configuration)
```
Comment (maciejSamerdak) [inline README.md]: tenor-2

**Pull request merged**